docs/linting/Troubleshooting.mdx

```diff
@@ -249,3 +249,9 @@ Rules such as [`no-unsafe-argument`](https://typescript-eslint.io/rules/no-unsaf
 ## My linting feels really slow
 
 If you think you're having issues with performance, see our [Performance Troubleshooting documentation](./troubleshooting/Performance.md).
+
+## Is TypeScript project references supported?
+
+No, TypeScript project references is not yet supported. 
+
+See [this issue](https://github.com/typescript-eslint/typescript-eslint/issues/2094) for more details.
```

Comment:
[Accessibility] Folks navigating through the page through the link tree wouldn't glean much context from just _"this issue"_. https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html

```suggestion
See [this issue](https://github.com/typescript-eslint/typescript-eslint/issues/2094 "typescript-eslint issue 2094 (project references support)") for more details.
```

Reply:
@JoshuaKGoldberg This syntax creates a `title` for the `<a>` link. Do you have sources that `title` helps with SRs? My past learnings have told me otherwise, e.g. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns)

My suggestion would just be to expand the link area to include more context.

Reply:
Hmm, maybe I'm wrong here then 😄